socorro/unittest/external/es/test_crashstorage.py

```diff
@@ -615,3 +612,215 @@ def esindex_fn(*args, **kwargs):
             body=document,
             **additional
         )
+
+    @mock.patch('socorro.external.es.connection_context.elasticsearch')
```

Comment:
Because that doesn't work here. I do not know why. ``mock`` is black magic to me, I usually just try patterns until I find one that works. This one works, the ``@mock.patch('elasticsearch.Elasticsearch')`` doesn't, so I'm sticking to mine. :)

However, if you want to educate me on the art of mocking and can explain what I am doing wrong, I would gladly learn!